Frends.ServiceBus/ServiceBus.cs

```diff
@@ -341,6 +348,109 @@ private static Encoding GetEncodingFromContentType(string contentTypeString, Enc
                 }
             }
         }
+
+        static async public Task<ReadResult> SendAndWaitForResponse(SendAndWaitForResponseInput input, SendAndWaitForResponseOptions options, CancellationToken token = new CancellationToken())
+        {
+            // Prepare queues
+            if (options.CreateQueues)
+            {
+                await EnsureQueueExists(input.Queue, input.ConnectionString, false);
+                await EnsureQueueExists(input.ReplyToQueue, input.ConnectionString, true);
+            }
+
+            QueueClient requestClient = MessagingFactory.CreateFromConnectionString(input.ConnectionString).CreateQueueClient(input.Queue);
```

Comment:
Creating the MessagingFactory is really slow, so this should use the ServiceBusMessagingFactory, preferably the cached instance to initialize the queue clients.

Furthermore, I guess the response could be sent to a topic as well, so the client should then be a SubscriptionClient